src/Hooks/Database/Hooks/DatabaseMutation.php

```diff
@@ -43,6 +44,15 @@ public function fires(): bool
         return Arr::has($this->mapping, get_class($this->event));
     }
 
+    public function for($event)
+    {
+        parent::for($event);
+
+        event(new ConfigureDatabaseMutation($event, $this));
```

Comment:
Why under for() and not fire()? This looks out of place.

Reply:
Same as we did with the Migrations & Seeds hooks :)

Reply:
That's not how it was intended. You should use Tenancy\Pipeline\Events\Resolving for that.

Reply:
In that case, we can simply get rid of this.

Reply:
Fixed